lib/modules/platform/gerrit/index.md

```diff
@@ -0,0 +1,72 @@
+# Gerrit
+
+## Supported Gerrit Versions
+
+All Gerrit 3.x Versions should be working.
+
+The current implementation uses the "hashtags" Feature from Gerrit and therefore needs a Gerrit version with [NoteDB](https://gerrit-review.googlesource.com/Documentation/note-db.html) backend.
+Gerrit `2.x` with NoteDB (only in `2.15` and `2.16`) is not tested, but could work.
+
+## Authentication
+
+<figure markdown>
+  ![Gerrit HTTP access token](/assets/images/gerrit-http-password.png){ loading=lazy }
+  <figcaption>First, create a HTTP access token for the Renovate account.</figcaption>
+</figure>
+
+Let Renovate use your HTTP access token by doing _one_ of the following:
+
+- Set your HTTP access token as a `password` in your `config.js` file
+- Set your HTTP access token as an environment variable `RENOVATE_PASSWORD`
+- Set your HTTP access token when you run Renovate in the CLI with `--password=`
```

Comment:
```suggestion
- Set your HTTP access token as a `password` in your `config.js` file, or
- Set your HTTP access token as an environment variable `RENOVATE_PASSWORD`, or
- Set your HTTP access token when you run Renovate in the CLI with `--password=`
```

I hope that adding `,or` to the end of the items make it clearer that you should use _one_ of these methods.